charts/velero/Chart.yaml

```diff
@@ -18,4 +18,3 @@ maintainers:
     email: brubakern@vmware.com
   - name: skriss
     email: krisss@vmware.com
-tillerVersion: ">=2.10.0"
```

Comment:
Remove this won't harm helm v2?

Reply:
this is optional, and I dont think users are using helm v2 under 2.9 :)

Reply:
but removed that, because the chart-testing removed that from the lint schema, so if we keep the lint will fail